source/hardware/controllers/gemini_g4v.rst

```diff
@@ -0,0 +1,250 @@
+Gemini G4V
+=====================
+
+.. figure:: ../../_static/controllers/gemini_g4v_topview.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+The Gemini G4V is a 2 deck controller that supports 4 virtual decks, a 4 way mixer and has a built-in 4 channel USB soundcard.
+It includes a Microphone input line, an Aux input line and an Booth output line. This lines are not going through the soundcard but mixed directly at the controller.
+They are not accessible nor controlled by Mixx so they will not appear in any recording nor is possible to control them via the Mixxx screen interface.
+
+
+-  `Manufacturer’s product page <http://geminisound.com/product/g4v>`__
+-  `Forum thread <https://www.mixxx.org/forums/viewtopic.php?f=6&t=12919>`__
+-  `Manufacturer’s User Guide <https://www.manualslib.com/manual/826563/Gemini-G4v.html>`__
+
+.. versionadded:: 2.3
+
+Mixxx sound hardware preferences
+--------------------------------
+
+========== =============
+Output     Channel
+========== =============
+Master     Channel 1 - 2
+Headphones Channel 3 - 4
+========== =============
+
+Mapping Description
+-------------------
+
+Most of the board's controls work as described in the G4V's manual, with a few exceptions. Snapshots of the board's labeling from the Manual are included below:
+
+
+.. figure:: ../../_static/controllers/gemini_g4v_controls.png
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+.. figure:: ../../_static/controllers/gemini_g4v_frontview.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+.. figure:: ../../_static/controllers/gemini_g4v_controllist.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+Shift
+-----
+
+On each of the physical decks is a :hwlabel:`SHIFT` button.
+
+This button, when held, changes the behavior of various buttons on that deck, labeled under the respective button.
+
+Mixer
+-----
+
+In the middle of board, there are :hwlabel:`HIGH`, :hwlabel:`MID`, and :hwlabel:`LOW` equalizer knobs, :hwlabel:`GAIN`, and :hwlabel:`FILTER` knobs for the individual virtual decks that work as normal.
+
+The :hwlabel:`MASTER` and :hwlabel:`CUE MIX` knobs works as usual.
+
+The :hwlabel:`BOOTH` knob controls the gain of the booth audio output. This output is managed internally by the controller.
+
+Crossfader
+----------
+
+The :hwlabel:`CROSSFADER` works as usual.
+
+The numbered buttons on the sides of the crossfader change the orientation of the virtual decks with respect to the crossfader.
+
+Deck Select Buttons
+-------------------
+
+Just below the Tempo slider on either side of the board is a :hwlabel:`Deck Sel` button (1|3 on the left and 2|4 on the right).
```

Comment:
```suggestion
Just below the Tempo slider on either side of the board is a :hwlabel:`DECK SEL` button (:hwlabel:`C|A` on the left and :hwlabel:`B|D` on the right).
```

Reply:
Why to change 1|3 (the actual label in the controller) to C|A ?

Reply:
Sorry, I used a wring image of thd controller for reference: https://idmmag.com/wp-content/uploads/2013/01/Gemini-G4V.jpg

Now I see that thf label is different on the actual controller.

Reply:
Ouch. Yes, this is really confusing.
Don't worry. I contested to the only two that where really relevant.
I appreciate your comments and agree that they increase readability. Any other comment will be welcomed.
I think that now the quality is quite good.
Is there anything else on my side to be done?
Thanks.
FYI. I didn't pay too much attention to the case because the hwlabel tag already makes the text to become upper case in the final document.

Reply:
I'll review again using the correct labels. Maybe we need to revert some of my proposed changes, sorry.

Reading my comment from above again, I feel it's necessary to point that I was not drunk when I commented, it was just late and I wrote the comment my phone :D